content/en/tracing/guide/ingestion_sampling_use_cases.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Trace Sampling Use Cases
+kind: guide
+further_reading:
+- link: "/tracing/guide/trace_ingestion_volume_control/"
+  tag: "Guide"
+  text: "How to control ingested volumes"
+---
+
+## Overview
+
+Trace data tends to be repetitive. A problem in your application is rarely identified in a single trace. For high throughput services, particularly for incidents that require your attention, an issue shows symptoms in multiple traces. Consequently, there’s usually no need for you to collect every single trace for a service or endpoint. Datadog APM [ingestion control mechanisms][1] help you keep the visibility that you need to troubleshoot problems while cutting down the noise and managing costs.
+
+This guide helps you understand when and how to use ingestion control configurations depending on the main use cases you might encounter. 
+
+Ingestion mechanisms are controllable from the Datadog Agent and from Datadog tracing libraries. If you are using OpenTelemetry SDKs to instrument your applications, read [Ingestion Sampling with OpenTelemetry][2].
+
+## Knowing which ingestion mechanisms are used
+
+To identify what ingestion mechanisms are currently used in your Datadog environment, navigate to the [Ingestion Control Page][3].
+
+{{< img src="/tracing/guide/ingestion_sampling_use_cases/ingestion_control_page.png" alt="Ingestion Control Page" style="width:90%;" >}}
+
+The table gives insights on ingested volumes *by service*. The Configuration column provides a first indication of the current set up. It shows: 
+- `AUTOMATIC` if the sampling rate calculated in the Datadog Agent is applied to the traces that start from the service. Read more about the specifics of [Datadog Agent ingestion logic][5].
+- `CONFIGURED` if a custom sampling rate configured in the tracing library is applied to the traces that start from the service.
+
+Click on various services to see details about what sampling decision makers are used for each service, as well as what [ingestion sampling mechanisms][1] are leveraged for ingested spans' services. 
```

Comment:
Here we reference `sampling decision makers`, but there is no glossary/explanation on what a decision maker is. So, someone new to Datadog concepts will not really know what this is. At least it sounded weird to me (and still sometimes is :) ) when I joined.

Reply:
I added a little clarification :)